client/stake/lcd/query.go

```diff
@@ -68,7 +68,7 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Co
 
 	// Get all validators
 	r.HandleFunc(
-		"/stake/validators",
+		"/stake/validators/{page}/{size}",
```

Comment:
Move the page and pagesize out of the URL. They should be the params instead of the restful URL.